cloud import: add descriptions about access keys (#12448)
pingcap/docs: tidb-cloud/import-sample-data.md

    - **Data format**: select **SQL File**.
    - **Bucket URI**: enter the sample data URI `s3://tidbcloud-sample-data/data-ingestion/`

>     - **Bucket Access**: to import the sample data, you can only use a Role ARN to access your bucket.

Reviewer: ```suggestion
    - **Bucket Access**: for sample data, you can only use a Role ARN to access its bucket. For your own data, you can use either an AWS access key or a Role ARN to access your bucket.
```

>         - **AWS Access Keys**: TiDB Cloud does not support using this method when you import the sample data.

Reviewer: ```suggestion
        - **AWS Access Keys**: Skip this option for the sample data.
```

>         - **AWS Access Keys**: enter the access key ID and secret access key. For more information about configuring access keys, see [Configure Amazon S3 access](/tidb-cloud/config-s3-and-gcs-access.md#configure-amazon-s3-access).

Reviewer: Do we have the available AWS Access Keys for the sample data at s3://tidbcloud-sample-data/data-ingestion/? If not, we can tell users to skip this option for sample data?

Author: @okJiang PTAL

Reviewer 2: The sample-data is not available by any Access Keys. I think this is a new feature we have not completed.

Author: updated
